docs/resources/dms_rocketmq_user.md

```diff
@@ -0,0 +1,87 @@
+---
+subcategory: "Distributed Message Service (DMS)"
+---
+
+# huaweicloud_dms_rocketmq_user
+
+Manages DMS RocketMQ user resources within HuaweiCloud.
+
+## Example Usage
+
+```HCL
+variable "instance_id" {}
+
+resource "huaweicloud_dms_rocketmq_user" "test" {
+  instance_id          = var.instance_id
+  access_key           = "user_test"
+  secret_key           = "abcdefg"
+  white_remote_address = "10.10.10.10"
+  admin                = false
+  default_topic_perm   = "PUB"
+  default_group_perm   = "PUB"
+  
+  topic_perms {
+    name = "topic_name"
+    perm = "PUB"
+  }
+  
+  group_perms {
+    name = "group_name"
+    perm = "PUB"
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) Specifies the region in which to create the resource.
+  If omitted, the provider-level region will be used. Changing this parameter will create a new resource.
+
+* `instance_id` - (Required, String, ForceNew) Specifies the ID of the rocketMQ instance.
+  Changing this parameter will create a new resource.
+
+* `access_key` - (Required, String, ForceNew) Specifies the access key of the user.
+  Changing this parameter will create a new resource.
+
+* `secret_key` - (Required, String, ForceNew) Specifies the secret key of the user.
+  Changing this parameter will create a new resource.
+
+* `white_remote_address` - (Optional, String) Specifies the IP address whitelist.
+
+* `admin` - (Optional, Bool) Specifies whether the user is an administrator.
+
+* `default_topic_perm` - (Optional, String) Specifies the default topic permissions.
+  Value options: **PUB|SUB**, **PUB**, **SUB**, **DENY**.
+
+* `default_group_perm` - (Optional, String) Specifies the default consumer group permissions.
+  Value options: **PUB|SUB**, **PUB**, **SUB**, **DENY**.

```

Comment:
please add the description of the default value.

Reply:
done